pyensembl/gene.py

```diff
@@ -73,6 +74,14 @@ def __eq__(self, other):
     def __hash__(self):
         return hash(self.id)
 
+    def __getstate__(self):
+        # Must be in order of __init__ arguments
+        return [self.id, self.name, self.contig, self.start, self.end, self.strand,
```

Comment:
I don't know if this is easier or more maintainable, but you can also return a dictionary here (which at least lets us later reorder constructor args). Not a big deal though.